functions/concepts/index.js

```diff
@@ -104,3 +104,18 @@ exports.makeRequest = async (req, res) => {
   res.sendStatus(externalRes.ok ? 200 : 500);
 };
 // [END functions_concepts_requests]
+
+exports.nodeTermination = async (event, context, callback) => {
+  // [START functions_concepts_node_termination]
```

Comment:
What region tag is this? Is this a new sample?

Reply:
Yes - see b/176872447